routers.go

```diff
@@ -176,30 +224,31 @@ func initWeb(conf cloudServerConfig) *webAPI {
 	}
 
 	// Initialize minio client for AWS Signature Version '4'
-	inSecure := !conf.TLS // Insecure true when TLS is false.
-	client, e := minio.NewV4(net.JoinHostPort(host, port), conf.AccessKeyID, conf.SecretAccessKey, inSecure)
+	insecure := !conf.TLS // Insecure true when TLS is false.
+	client, e := minio.NewV4(net.JoinHostPort(host, port), conf.AccessKeyID, conf.SecretAccessKey, insecure)
 	fatalIf(probe.NewError(e), "Unable to initialize minio client", nil)
 
-	w := &webAPI{
+	return &webAPI{
 		FSPath:          conf.Path,
 		AccessLog:       conf.AccessLog,
 		Client:          client,
 		apiAddress:      conf.Address,
+		insecure:        insecure,
 		accessKeyID:     conf.AccessKeyID,
 		secretAccessKey: conf.SecretAccessKey,
+		storageapi:      storageapi,
```

Comment:
I am assuming this change is for prepping for 'minio-go' removal. But you shouldn't put the whole storage handlers inside web handlers it is not meant to be done this way. 

web-handlers should access FS itself directly which was initialized for 'API' as well.